doc/config.toml

```diff
@@ -0,0 +1,22 @@
+baseURL = "http://localhost"
```

Comment:
Is this going right?

Reply:
Yes, this is for local builds. We override it in the `Makefile` when generating the doc.